containers/payments/BillingSection.js

```diff
@@ -173,7 +173,7 @@ const BillingSection = ({ permission }) => {
                             </div>
                         </div>
                     ) : null}
-                    {hasPaidVpn ? (
```

Comment:
maybe my user was corrupted somehow, but It crashed because vpnPlan was undefined, but hasPaidVpn=4